src/AccessibilityInsights.Extensions.Telemetry/OSHelpers.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.Win32;
+
+namespace AccessibilityInsights.Extensions.Telemetry
+{
+    /// <summary>
+    /// Helpers to extract data from the OS
+    /// </summary>
+    internal static class OSHelpers
+    {
+        /// <summary>
```

Comment:
I still wish there were a remark as to why this is suitable for telemetry but not for features. There's no "why".

Reply:
There are countless blog pages on the problems with the various forms of GetVersion. I don't see a lot of value in adding yet another commentary.